src/tools/altinn-fleet-stats/backend/altinn_fleet/fetcher.py

```
@@ -0,0 +1,248 @@
+            return "updated"
+
+        target.parent.mkdir(parents=True, exist_ok=True)
+        rc, _, err = await self._git("clone", "-q", repo_url, str(target))
+        if rc != 0:
+            target.mkdir(parents=True, exist_ok=True)
+            failed_marker.write_text(f"clone failed: {err}\n")
+            return "failed"
+        if commit:
+            rc, _, err = await self._git("checkout", "-q", commit, cwd=target)
+            if rc != 0 and "unable to read tree" in err:
+                rc2, branch_out, _ = await self._git("remote", "show", "origin", cwd=target)
```

Ruff (0.15.12)

[warning] Unpacked variable `rc2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)